insights/parsers/ip_netns_exec_namespace_lsof.py

```diff
@@ -0,0 +1,82 @@
+"""
+IpNetnsExecNamespaceLsofI - command ``/sbin/ip netns exec [network-namespace] lsof -i"``
+========================================================================================
+
+This module provides class ``IpNetnsExecNamespaceLsofI`` for parsing the output of command
+``/sbin/ip netns exec [network-namespace] lsof -i``.
+Filters have been added so that sensitive information can be filtered out.
+This results in the modification of the original structure of data.
+"""
+
+from collections import namedtuple
+from insights import CommandParser, parser
+from insights.core.filters import add_filter
+from insights.parsers import SkipException, keyword_search, parse_fixed_table
+from insights.specs import Specs
+
+FILTERS = ["COMMAND", "neutron-n"]
```

Comment:
The `neutron-n` should be moved to plugins.

Reply:
@xiangce - Acknowledged. Changes done in the latest commit.